tasks/clone.js

```diff
@@ -6,15 +6,15 @@ module.exports = function(grunt) {
   var templateHelper = require('./lib/templateHelper').init(grunt);
   var _ = require('underscore');
 
-  function cloneAppTemplates(filepath, cloneDepth) {
+  function cloneAppTemplates(filepath, cloneDepth, useSsh) {
     grunt.config.set('gitclone', {});
 
     var globalJson = grunt.file.readJSON(filepath);
 
     templateHelper.walkObjects(globalJson.show, function (obj) {
       var prefix = 'gitclone.' + obj.id + '.options';
       if (_.has(obj, 'repoUrl') && _.has(obj, 'repoBranch')) {
-        var repoUrl = obj.repoUrl.replace('git://github.com/','git@github.com:');
+        var repoUrl = useSsh ? obj.repoUrl.replace('git://github.com/','git@github.com:') : obj.repoUrl;
```

Comment:
It might be possible to just say `var repoUrl = obj.repoUrl` here now (and remove the ssh configurability), but I'm not sure. What I'm thinking is that it could have been originally set to SSH because there were private templates being cloned. Does that sound likely?